wled00/overlay.cpp

```diff
@@ -89,6 +89,15 @@ void _overlayAnalogCountdown()
 
 void handleOverlayDraw() {
   usermods.handleOverlayDraw();
+  if (analogClockSolidBlack) {
+    const Segment* segments = strip.getSegments();
+    for (uint8_t i = 0; i < strip.getActiveSegmentsNum(); i++) {
```

Comment:
This may be incorrect as you are iterating across raw `segments[]` vector/array.
The correct way would be:
```c++
for (int i=0; i<strip.getSegmentsNum(); i++) {
  const Segment &segment = segments[i];
  if (!segment.isActive()) continue;
...
}
```

Reply:
Sure you are right!
Sorry for that! 😞